src/Mono.Android/Mono.Android.targets

```diff
@@ -285,6 +287,7 @@
     <PropertyGroup>
       <_Binlog>$(MSBuildThisFileDirectory)../../bin/Build$(Configuration)/UpdateExternalDocumentation-$([System.DateTime]::Now.ToString ("yyyyMMddTHHmmss")).binlog</_Binlog>
     </PropertyGroup>
+    <Exec Command="dotnet build -v:n -c $(Configuration) $(JavaInteropFullPath)/build-tools/jnienv-gen/jnienv-gen.csproj" />
```

Comment:
I don't understand this change.  Why do we need it?  If we *do* need it, shouldn't we instead add `jnienv-gen.csproj` to `@(ProjectReference)` in `Mono.Android.csproj`?

Reply:
This was needed when building from a fully clean state, I've added the project reference to Mono.Android.csproj instead to see if that will work.

Reply:
I'm just going to back this change out for now as the proper fix may warrant a java.interop changes as well.